extensions/wikia/ArticleVideo/bower_components/html5-skin/js/controller.js

```diff
@@ -0,0 +1,1496 @@
+      if (this.state.buffering === true) {
+        this.state.buffering = false;
+        this.renderSkin();
+      }
+    },
+
+    onReplay: function(event) {
+      this.resetUpNextInfo(false);
+    },
+
+    onAssetDimensionsReceived: function(event, params) {
+      if (params.videoId == OO.VIDEO.MAIN && (this.skin.props.skinConfig.responsive.aspectRatio == "auto" || !this.skin.props.skinConfig.responsive.aspectRatio)) {
+        this.state.mainVideoAspectRatio = this.calculateAspectRatio(params.width, params.height);
+        this.setAspectRatio();
+      }
+    },
+
+    /********************************************************************
+      ADS RELATED EVENTS
+    *********************************************************************/
+
+    onAdsPlayed: function(event) {
+      OO.log("onAdsPlayed is called from event = " + event);
+      this.state.screenToShow = CONSTANTS.SCREEN.PLAYING_SCREEN;
+      this.skin.updatePlayhead(this.state.mainVideoPlayhead, this.state.mainVideoDuration, this.state.mainVideoBuffered);
+      this.state.duration = this.state.contentTree.duration / 1000;
+      this.state.isPlayingAd = false;
+      this.state.pluginsElement.removeClass("oo-showing");
+      this.state.pluginsClickElement.removeClass("oo-showing");
+      this.renderSkin();
+    },
+
+    onWillPlayAds: function(event) {
+      OO.log("onWillPlayAds is called from event = " + event);
+      this.state.isPlayingAd = true;
+      this.state.pluginsElement.addClass("oo-showing");
+      this.state.pluginsElement.css({
+        height: "",
+        width: ""
+      });
+      this.state.forceControlBarVisible = (this.state.pluginsElement.children().length > 0);
+    },
+
+    onAdPodStarted: function(event, numberOfAds) {
+      OO.log("onAdPodStarted is called from event = " + event + " with " + numberOfAds + " ads");
+      this.state.currentAdsInfo.numberOfAds = numberOfAds;
+      this.renderSkin();
+    },
+
+    onWillPlaySingleAd: function(event, adItem) {
+      OO.log("onWillPlaySingleAd is called with adItem = " + adItem);
+      if (adItem !== null) {
+        this.state.adVideoDuration = adItem.duration;
+        this.state.screenToShow = CONSTANTS.SCREEN.AD_SCREEN;
+        this.state.isPlayingAd = true;
+        this.state.currentAdsInfo.currentAdItem = adItem;
+        this.state.playerState = CONSTANTS.STATE.PLAYING;
+        if (adItem.isLive) {
+          this.state.adStartTime = new Date().getTime();
+        } else {
+          this.state.adStartTime = 0;
+        }
+        this.skin.state.currentPlayhead = 0;
+        this.state.mainVideoElement.removeClass('oo-blur');
+        this.renderSkin();
+      }
+    },
+
+    onSingleAdPlayed: function(event) {
+      OO.log("onSingleAdPlayed is called");
+      this.state.isPlayingAd = false;
+      this.state.adVideoDuration = 0;
+      this.state.currentAdsInfo.skipAdButtonEnabled = false;
+    },
+
+    onShowAdSkipButton: function(event) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "event". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)